jbehave-support-core/src/main/java/org/jbehavesupport/core/rest/RestServiceHandler.java

```diff
@@ -125,6 +127,43 @@ public void sendRequest(String urlPath, HttpMethod requestMethod, ExamplesTable
         } catch (HttpClientErrorException | HttpServerErrorException e) {
             storeResponse(e);
         }
+        isSuccessful();
+    }
+
+    private void isSuccessful() {
+        if (testContext.keySet().containsAll(
+            Arrays.asList(REST_RESPONSE_CODE, REST_RESPONSE_HEADERS, REST_RESPONSE_JSON))) {
+            isSuccessful(HttpStatus.valueOf((Integer)testContext.get(REST_RESPONSE_CODE)),
+                testContext.get(REST_RESPONSE_HEADERS),
+                testContext.get(REST_RESPONSE_JSON));
+        } else {
+            log.warn("Test context doesn't have consistent REST response stored, skipping success handler");
+        }
+    }
+
+    /**
+     * Custom verifier for each response
+     * Method is internally called by API method {@link RestServiceHandler#sendRequest}.
+     */
+    public void isSuccessful(HttpStatus responseCode, HttpHeaders responseHeaders, Object response){
+
+    }
+
+    /**
+     * Each response that meant to be successful must match status
+     * @return HttpStatus that represents success result
+     */
+    public HttpStatus getSuccessStatus(){
+        return null;
```

Comment:
IMHO it might be good to provide default implementation with 2xx range of statuses, what do you think?

Reply:
Range is not supported.

Reply:
Not sure I understand - this is our method so we could support ranges if we would want to, right?

Reply:
Feel free to show POC.

Reply:
Ok, I see we got our hands tied by `public void verifyResponse(String expectedStatus, ExamplesTable data)`:slightly_frowning_face: 
In that case I still propose to have `HttpStatus.OK` as a default - what do others think?